Index: src/app/page.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// src/app/page.tsx\r\nimport Footer from '@/components/footer';\r\nimport Header from '@/components/header';\r\nimport HeroSection from '@/components/hero-section';\r\nimport ProductShowcase from '@/components/productos';\r\nimport PromoSouvenirComponent from '@/components/promo-souvenir';\r\nimport SocialMediaIcons from '@/components/socials';\r\nimport TestimonialsComponent from '@/components/testimonials';\r\nimport React from 'react';\r\n\r\n\r\nexport default function Home() {\r\n  return (\r\n    <main className=\"min-h-screen flex flex-col\">\r\n      <Header />\r\n      <HeroSection/>\r\n      <ProductShowcase/>\r\n      <PromoSouvenirComponent/>\r\n      <TestimonialsComponent/>\r\n      <SocialMediaIcons/>\r\n      <Footer/>\r\n    </main>\r\n  );\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/app/page.tsx b/src/app/page.tsx
--- a/src/app/page.tsx	(revision 0193f38d9606b0aef9b671261221551c2aeda3a8)
+++ b/src/app/page.tsx	(date 1724817890301)
@@ -7,17 +7,15 @@
 import SocialMediaIcons from '@/components/socials';
 import TestimonialsComponent from '@/components/testimonials';
 import React from 'react';
+import SignUp from "@/components/pages/sign-up";
 
 
 export default function Home() {
   return (
     <main className="min-h-screen flex flex-col">
       <Header />
-      <HeroSection/>
-      <ProductShowcase/>
-      <PromoSouvenirComponent/>
-      <TestimonialsComponent/>
-      <SocialMediaIcons/>
+      <SignUp/>
+
       <Footer/>
     </main>
   );
Index: .idea/inspectionProfiles/Project_Default.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/inspectionProfiles/Project_Default.xml b/.idea/inspectionProfiles/Project_Default.xml
new file mode 100644
--- /dev/null	(date 1724816962969)
+++ b/.idea/inspectionProfiles/Project_Default.xml	(date 1724816962969)
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="Eslint" enabled="true" level="WARNING" enabled_by_default="true" />
+  </profile>
+</component>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1724816963104)
+++ b/.idea/vcs.xml	(date 1724816963104)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1724816963197)
+++ b/.idea/.gitignore	(date 1724816963197)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
Index: .idea/tarjetas-con-amor.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/tarjetas-con-amor.iml b/.idea/tarjetas-con-amor.iml
new file mode 100644
--- /dev/null	(date 1724816963073)
+++ b/.idea/tarjetas-con-amor.iml	(date 1724816963073)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1724816963073)
+++ b/.idea/modules.xml	(date 1724816963073)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/tarjetas-con-amor.iml" filepath="$PROJECT_DIR$/.idea/tarjetas-con-amor.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: src/components/pages/sign-up.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/pages/sign-up.tsx b/src/components/pages/sign-up.tsx
new file mode 100644
--- /dev/null	(date 1724837358970)
+++ b/src/components/pages/sign-up.tsx	(date 1724837358970)
@@ -0,0 +1,96 @@
+// pages/index.js
+import Image from 'next/image';
+
+export default function SignUp() {
+    return (
+        <div className="relative flex flex-col min-h-screen">
+            {/* Imagen de fondo */}
+            <div className="absolute inset-0 -z-0">
+                <Image
+                    src="/fondo.jpg"
+                    alt="Descripción de la imagen"
+                    layout="fill"
+                    objectFit="cover"
+                    objectPosition="center center"
+                />
+            </div>
+
+            {/* Contenedor para las secciones de formulario e información */}
+            <div className="relative z-10 flex flex-col md:flex-row p-4 md:p-8 justify-center space-x-0">
+                {/* Sección del formulario */}
+                <div className="border border-white p-4 w-full max-w-md bg-black bg-opacity-70">
+                    <h2 className="text-2xl dark:text-white font-geometos">Empieza a crear memorias</h2>
+
+                    <form className="grid grid-cols-2 gap-4 mt-4">
+                        <div>
+                            <label className="block text-white" htmlFor="name">Nombre</label>
+                            <input
+                                className="w-full p-2"
+                                type="text"
+                                id="name"
+                            />
+                        </div>
+                        <div>
+                            <label className="block text-white" htmlFor="apellido">Apellido</label>
+                            <input
+                                className="w-full p-2"
+                                type="text"
+                                id="apellido"
+                            />
+                        </div>
+                        <div>
+                            <label className="block text-white" htmlFor="email">Correo (Required)</label>
+                            <input
+                                className="w-full p-2"
+                                type="email"
+                                id="email"
+                            />
+                        </div>
+                        <div>
+                            <label className="block text-white" htmlFor="confirmEmail">Confirma tu correo</label>
+                            <input
+                                className="w-full p-2"
+                                type="email"
+                                id="confirmEmail"
+                            />
+                        </div>
+                        <div>
+                            <label className="block text-white" htmlFor="password">Contraseña (Required)</label>
+                            <input
+                                className="w-full p-2"
+                                type="password"
+                                id="password"
+                            />
+                        </div>
+                        <div>
+                            <label className="block text-white" htmlFor="confirmPassword">Confirma tu contraseña</label>
+                            <input
+                                className="w-full p-2"
+                                type="password"
+                                id="confirmPassword"
+                            />
+                        </div>
+                        <div className="col-span-2">
+                            <button
+                                className="w-full bg-blue-500 text-white p-2 rounded-lg hover:bg-blue-600 transition"
+                                type="submit"
+                            >
+                                Enviar
+                            </button>
+                        </div>
+                    </form>
+                </div>
+
+                {/* Sección del contenedor de información */}
+                <div className="border border-white p-4 w-full max-w-md bg-white bg-opacity-70">
+                    <h2 className="text-xl font-bold mb-4">Información</h2>
+                    <p>
+                        Aquí puedes poner la información relevante que quieras mostrar. Esta sección es
+                        completamente flexible y puede contener cualquier tipo de contenido que desees,
+                        como textos, enlaces, imágenes, etc.
+                    </p>
+                </div>
+            </div>
+        </div>
+    );
+}
